controller/space_iterations.go

```diff
@@ -115,8 +115,8 @@ func (c *SpaceIterationsController) Create(ctx *app.CreateSpaceIterationsContext
 // List runs the list action.
 func (c *SpaceIterationsController) List(ctx *app.ListSpaceIterationsContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		_, err := appl.Spaces().Load(ctx, ctx.SpaceID)
-		if err != nil {
+		exists, err := appl.Spaces().Exists(ctx, ctx.SpaceID.String())
+		if !exists || err != nil {
```

Comment:
We should handle the `err` separately as the `repository.Exists` implementation might return an `InternalError`.  This is probably true for every call of this the Exists function.